scripts/solutions.sh

```diff
@@ -261,6 +259,11 @@ configure_archives() {
     local removed=${1:-False}
 
     for archive in "${ARCHIVES[@]}"; do
+        if file "$archive" | grep -vq 'ISO 9660'; then
```

Comment:
Minor and can be omitted
We are searching for a fixed string here `ISO 9660` so we can make this faster with `fgrep`?? no??

Reply:
Or `grep -F` (`fgrep` is deprecated, as explained in the man page).

But I don't think we will observe a perf improvement here ^^